scripts/recreating.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# This script will build and run OpenRegistry, then build and run
+# the OCI conformance tests against it.
+# It depends on bash, docker and git being available.
+
+set -eu
+
+spec_name=distribution-spec
+spec_version=v1.0.1
+prod_name=example
+trow_version=v0.2.0
+OPENREGISTRY_IMAGE_NAME="openregistry-distribution-build:v$(date +%Y%m%d%H%M%S)"
+
+# check out to OpenRegistry repo
+rm -rf OpenRegistry && git clone git@github.com:containerish/OpenRegistry.git
+pushd OpenRegistry
+docker build -f ./Dockerfile -t "${OPENREGISTRY_IMAGE_NAME}" .
+
+IP=$(ifconfig | grep 192 | awk '{print $2}')
```

Comment:
@guacamole this command doesn't work well with linux, let's resuse the commented version of this command